test/e2e/scalability/density.go

```diff
@@ -275,7 +299,10 @@ func cleanupDensityTest(dtc DensityTestConfig) {
 			framework.ExpectNoError(err)
 		}
 	}
+	podCleanupPhase.End()
 
+	auxCleanupPhase := testPhaseDurations.StartPhase(910, "secrets, config maps and daemon sets cleanup")
```

Comment:
No space in configmaps. Also maybe s/cleanup/deletion/ ?

Reply:
No space for daemon sets too.

Reply:
Done.